### ISSUE TITLE
Executing `test_scl_usage` when tty exists waits on 'cat' forever

### ISSUE THREAD

**Issue by lukaszachy** [sclorg/s2i-php-container#387]
From bash verbose when this test is executed:

```bash
[INFO] Testing the image SCL enable...

++ docker run --rm php-image /bin/bash -c ''
+ out=
+ echo ''
+ grep -q ''
+++ cat
```
and there it stops. When there is no tty cat exists immediately and test continues as expected.

**Comment by lukaszachy** [comment]
It looks that it happens because `cid_file` variable is no longer defined.

**Comment by lukaszachy** [comment]
@phracek Could you take a look please?

**Comment by phracek** [comment]
This issue looks like a regression of https://github.com/sclorg/container-common-scripts/pull/306

**Comment by pkubatrh** [comment]
This is still an issue with updated common scripts

**Comment by zmiklank** [comment]
This is fixed by ed62e9a8d5b30ee6906ca7a3a5fd47d046f65952.

It did not make sense to call this test explicitly from testsuite, as it is used as a part of other tests.

If issue still persists please feel free to reopen.
